docs/getting_started_guide/ISO_Build/Building_SIMP_From_Tarball.rst

```diff
@@ -63,16 +62,16 @@ For example:
 .. code::
 
    $ # for SIMP 6 and CentOS 7
-   $ bundle exec rake build:auto[.,6.X,SIMP-6.0.0-0-Overlay-EL-7-x86_64.tar.gz]
+   $ bundle exec rake build:auto[.,6.X,SIMP-6.1.0-0-Overlay-EL-7-x86_64.tar.gz]
```

Comment:
Did we test this? It was broken for quite some time.

Reply:
Getting feedback from those that did, and fixing